runtime/vm/CMakeLists.txt

```diff
@@ -124,6 +127,23 @@ add_library(j9vm SHARED
 	${CMAKE_CURRENT_BINARY_DIR}/ut_j9vm.c
 )
 
+# Interpreter sources may need special handling on some platforms
+set(interpreter_sources
+	BytecodeInterpreter.cpp
+	DebugBytecodeInterpreter.cpp
+	MHInterpreter.cpp
+)
+
+add_library(j9vm SHARED
+	${main_sources}
+	${interpreter_sources}
+)
+
+if(OMR_OS_AIX AND (OMR_TOOLCONFIG STREQUAL "xlc"))
```

Comment:
This applies to all PPC using XLC, not just AIX:
 https://github.com/eclipse/openj9/blob/d6314e979b0c20821c9150ad3a63b56a31396656/runtime/makelib/targets.mk.ftl#L549-L550